src/Umbraco.Core/Logging/Viewer/LogViewerSourceBase.cs

```diff
@@ -1,26 +1,35 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Xml;
 using Newtonsoft.Json;
 using Serilog.Events;
 using Umbraco.Core.IO;
 using Umbraco.Core.Models;
 using Umbraco.Core.Persistence.DatabaseModelDefinitions;
+using Formatting = Newtonsoft.Json.Formatting;
 
 namespace Umbraco.Core.Logging.Viewer
 {
     public abstract class LogViewerSourceBase : ILogViewer
     {
-        protected LogViewerSourceBase(string pathToSearches = "")
+        protected LogViewerSourceBase(string pathToSearches = "", string pathToSerilog = "")
         {
             if (string.IsNullOrEmpty(pathToSearches))
                 // ReSharper disable once StringLiteralTypo
                 pathToSearches = IOHelper.MapPath("~/Config/logviewer.searches.config.js");
 
             _searchesConfigPath = pathToSearches;
+
+            if (string.IsNullOrEmpty(pathToSerilog))
+                // ReSharper disable once StringLiteralTypo
+                pathToSerilog = IOHelper.MapPath("~/Config/serilog.config");
```

Comment:
I think instead, you should use `Umbraco.Core.Logging.Serilog.LoggerConfigExtensions.ReadFromConfigFile()` - this also helps not changing this method signature.

I'm not sure who would even need to give us the path of the serilog config? I see it's consistent with pathToSearches - but also not entirely sure why that's configurable.

Reply:
Ooh, @warrenbuckley suggested a possibly better way than reading and parsing config files, just ask Serilog if any of it's levels are enabled: https://stackoverflow.com/questions/49226533/how-to-get-current-serilog-minimumlevel

Reply:
That's much better - will update